### PR TITLE
Book Series & Books

### PR DESCRIPTION
**Pull request by rrrnld**

### CODE REVIEW

File: src/resources/books.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  copyright?: string
+  copyrightUrl?: string
+  license?: string
+  licenseCode?: string
+  licenseUrl?: string
+}
+
+export type Relationships = {
+  bookSeries: { type: BookSeries.Type; cardinality: '1' }
```
Comment (errnesto) [inline src/resources/books.ts]: I think a book might not be part of a series? For the frontend we said we make the ui like there was a series "without-series" is that how the data is handeled too?

Comment (rrrnld) [reply src/resources/books.ts]: You're right!

**Comment by errnesto** [comment]
I don't know what that supplementaty url is?
Other than that this lgtm :-)

**Comment by rrrnld** [comment]
Supplementary URLs are URLs that contain additional info on a book. Like `supplementary-*` files, but living somewhere else.

**Pull request merged**